### DISCUSSION TITLE
User type login system

### DISCUSSION THREAD

**Discussion by rgustprabha** [serverpod/serverpod#1354] [Q&A] [answered]
Hi how to use the user type login system in server pod, any suggestion would be appreciable for the following scenario

Example:

we have 3 type of user 
1. admin
2. manager
3. employee

How to specify the user type in the authentication module to make user type authentication.

**Comment by Frank1016** [comment, marked as answer]
Check this out:
https://docs.serverpod.dev/concepts/authentication/basics
You can set up user scopes for each user type:
![image](https://github.com/serverpod/serverpod/assets/32797520/aee208fb-e9a7-458b-b1dc-4d5fa50ae538)
